data/data-pipeline/data_pipeline/etl/score/constants.py

```diff
@@ -155,11 +157,11 @@
 
 # Finally we augment with the GEOID10, county, and state
 DOWNLOADABLE_SCORE_COLUMNS = [
-    "GEOID10_TRACT",
-    "County Name",
-    "State Name",
-    "Score G (communities)",
-    "Median household income (% of AMI)",
-    "Median household income (% of state median household income) (percentile)",
+    field_names.GEOID_TRACT_FIELD,
+    field_names.COUNTY_FIELD,
+    field_names.STATE_FIELD,
+    field_names.SCORE_G_COMMUNITIES,
```

Comment:
should this be `SCORE_L_COMMUNITIES` now?

Reply:
Ditto on above; happy to make changes but figured I'd wait til we have the final list of fields we want and where we want them.

Reply:
That makes a lot of sense!